1.Core/UIElements/Web/TextField.cs

```diff
@@ -22,17 +23,18 @@ public void Clear()
             }
             catch (ElementNotVisibleException error)
             {
-                System.Console.WriteLine($"Unable to visualize button: {Name}");
+                Logger.Instance.Error($"Unable to visualize {Name} button");
                 throw error;
             }
             catch (ElementNotInteractableException error)
             {
-                System.Console.WriteLine($"Unable to click button: {Name}");
+                Logger.Instance.Error($"Unable to interact with {Name} button");
+                ;
```

Comment:
Is this semi-colon in the right place?

Reply:
Actually no, but it's weird that the formatter didn't remove it. I already fixed it though.

Regarding the warnings, I'm solving that in another PR to not mix the issues.

Reply:
👍